src/content/operations/deployment/subctl/_index.en.md

```diff
@@ -530,3 +531,33 @@ The following steps are performed:
 | `--kubeconfig` `<string>`        | Absolute path(s) to the kubeconfig file(s)
 | `--namespace` `<string>`         | Namespace in which Submariner is installed (default `submariner-operator`)
 | `--yes`                          | Automatically answer yes to confirmation prompt
+
+### Running `subctl diagnose` from a Pod in cluster
+
+As of 0.12.0, a `subctl` image is provided for running the `subctl` binary from a Pod within a cluster. The output of
```

Comment:
```suggestion
As of 0.12.0, a [subctl](quay.io/submariner/subctl) image is provided for running the `subctl` binary from a Pod within a cluster. The output of
```

Since subctl comes in different forms, it would be good if we provide a link while referring to the subctl image.